src/AccessToken.php

```diff
@@ -395,7 +394,7 @@ private function retrieveCertsFromLocation($url, array $options = [])
      */
     private function checkAndInitializePhpsec()
     {
-        if (!$this->checkAndInitializePhpsec2() && !$this->checkPhpsec3()) {
+        if (!class_exists(RSA::class)) {
             throw new RuntimeException('Please require phpseclib/phpseclib v2 or v3 to use this utility.');
```

Comment:
This needs to be updated
```suggestion
            throw new RuntimeException('Please require phpseclib/phpseclib v3 to use this utility.');
```